### ISSUE TITLE
Property endpoints are not working anymore

### ISSUE THREAD

**Issue by tsathishkumar** [mozilla-iot/webthing-rust#7]
I tried to add a web thing in gateway and getting the below error,

```
Failed to connect to http://localhost:8888/0/properties/on : { FetchError: invalid json response body at http://localhost:8888/0/properties/on reason: Unexpected end of JSON input
```

The server isn't exposing the properties api's. 

Note: The endpoint was working on an older revision of webthing-rust ([b247100](https://github.com/mozilla-iot/webthing-rust/tree/b2471002bab8b0faab154dfd6ee323465af59487)).

**Comment by tsathishkumar** [comment]
Actix-web upgrade seems to have caused the issue. I'm working on the fix.

**Comment by mrstegeman** [comment]
Fixed by #8
